src/routes/activityReports/handlers.js

```diff
@@ -21,13 +22,9 @@ const logContext = {
  * @param {*} res - response
  */
 export async function getApprovers(req, res) {
-  const user = await userById(req.session.userId);
-  const reportRegions = user.permissions
-    .filter((p) => p.scopeId === READ_WRITE_REPORTS)
-    .map((p) => p.regionId);
-
+  const { region } = req.query;
```

Comment:
Should this method should be using the Users policy? I think there are a couple of methods still with no authorization, so I'm leaning towards not making any changes here beyond the good work you've already done, and just keeping #265 open for the other routes.

Reply:
:+1: that sounds good to me. There may be a few other routes to update that have been added in-between starting this work and getting it merged in to main. We'll probably just want to audit all our routes on main and make sure every route added after is using a policy (or explicitly doesn't)